ui/src/components/LandscapeWayfinding.tsx

```diff
@@ -0,0 +1,143 @@
+  const groupFlagInLocation = ob.isValidPatp(location.split('/')[4] || '');
+
+  return (
+    <div className="flex flex-col leading-5">
+      <h1 className="my-8 text-2xl font-bold">Where am I?</h1>
+      <p>
+        Tlon Corporation's "Groups" app, a multi-channel communications
+        platform.
+      </p>
+      {groupFlagInLocation && (
+        <p className="mt-4">
+          You're currently within a group, which might have a variety of
+          channels.
+        </p>
+      )}
+      <h1 className="my-8 text-2xl font-bold">What can I do here?</h1>
+      <p>
+        Groups is designed as a “channel-based communications” software, which
+        means that it works best when used with a group of people. You can join
+        existing communities, or start your own.{' '}
+      </p>
+      <p className="mt-4">
+        Here are some groups we recommend joining to learn more about Groups and
+        how to use it in interesting ways:
+      </p>
+      <div className="mt-8 space-y-2">
+        <GroupReference
+          flag="~halbex-palheb/uf-public"
+          plain
+          description="Learn about the Urbit Project"
+        />
+        <GroupReference
+          flag="~natnex-ronret/door-link"
+          description="A cult of music lovers"
+          plain
+        />
+        <GroupReference
+          flag="~nibset-napwyn/tlon"
+          description="A place to ask for help"
+          plain
+        />
+      </div>
+    </div>
+  );
+}
+
+function TalkDescription() {
+  return (
+    <div className="flex flex-col leading-5">
+      <h1 className="my-8 text-2xl font-bold">Where am I?</h1>
+      <p>
+        Tlon Corporation’s “Talk” app, a simple, powerful, and secure instant
+        messaging software for individuals or small groups of people.
+      </p>
+      <h1 className="my-8 text-2xl font-bold">What can I do here?</h1>
+      <p>
+        Talk is simple: It works like any other messaging app you’ve ever used.
+        What makes it special is its directly person-to-person nature, no one
+        person or company can ever snoop the messages you send on Talk.
+      </p>
+      <p className="mt-4 mb-8">
+        In addition to the experience you expect, Talk can also aggregate group
+        communications from Groups and other software in the Urbit ecosystem.
+      </p>
+    </div>
+  );
+}
+
+export default function LandscapeWayfinding() {
+  const [showModal, setShowModal] = useState(false);
+  const isMobile = useIsMobile();
+  const app = useAppName();
+  const gang = useGang('~nibset-napwyn/tlon');
+  const { open } = useGroupJoin('~nibset-napwyn/tlon', gang);
+
+  const handleHide = () => {
+    setCalmSetting('disableWayfinding', true);
+  };
+
+  return (
+    <Popover.Root>
+      <div
+        className={cn('absolute left-5 z-50', {
+          'bottom-10': !isMobile,
+          'bottom-20': isMobile,
+        })}
+      >
+        <Popover.Trigger className="relative" asChild>
```

Comment:
Any reason why popover instead of dropdown menu here?

Reply:
No, if you'd prefer we can switch to dropdown menu. I guess I just mentally categorize this as a "popover".